src/app/modules/calendar-week-hours-view/calendar-week-hours-view.component.ts

```diff
@@ -445,7 +447,7 @@ export class CalendarWeekHoursViewComponent implements OnChanges, OnInit, OnDest
             excluded: this.excludeDays,
             precision: this.precision,
             absolutePositionedEvents: true
-        });
+        }).eventRows;
```

Comment:
Is this modification really meant to be there? The compiler complains about this: Property 'eventRows' does not exist on type 'WeekViewEventRow[]'

Reply:
Actually yes...
I get this error when I remove this line:
Type 'WeekView' is not assignable to type 'WeekViewEventRow[]'

Reply:
Ok. It seems that the `getWeekView` method in `CalendarUtils` changed its signature between versions 0.22.3 and current (0.23.2). I think we need to update the peerDependency version for angular-calendar to make this change work.